src/components/LoginButton.js

```diff
@@ -0,0 +1,39 @@
+import PropTypes from 'prop-types';
+import { useCookies } from 'react-cookie';
```

Comment:
We don't I picked this one because it matches well with the hooks-philosophy of React. But I was also worried this will possibly have to be replaced by another package if we want to use the cookies in plain javascript files

Reply:
Not every function in a component needs to be a hook, what would be the advantage of using a hook here? Does it automatically handle internal state or re-renders?